src/widget/utils.js

```diff
@@ -89,6 +98,29 @@ export function getLabel( element ) {
 	return typeof labelCreator == 'function' ? labelCreator() : labelCreator;
 }
 
+/**
+ * Creates nested editable element with proper CSS classes.
+ *
+ * @param {String} elementName Name of the element to be created.
+ * @param {module:engine/view/document~Document} viewDocument
+ * @returns {module:engine/view/editableelement~EditableElement}
+ */
+export function createNestedEditable( elementName, viewDocument ) {
```

Comment:
It would be more consistent with `widgetize()` if it accepted an element.

BTW, it'd be hard to follow "widgetize"'s namig style in this case, so perhaps we could use these names: `toWidget()` and `toWidgetEditable()`.